R/distribution_formulas.R

```diff
@@ -0,0 +1,192 @@
+#' Set up a new distribution for a data type
+#'
+#' @param data_type A string specifying the type of data that the distribution
+#'   will be fit to. Options are listed in the function call, where the first
+#'   option listed, i.e., `"index"` is the default.
+#' @param module An identifier to a C++ fleet module that is linked to the data
+#'   of interest.
+#' @param family A description of the error distribution and link function to
+#'   be used in the model.
+#' @param sd A list of length two. The first entry, `"value"`, stores the
+#'   initial values for the relevant standard deviations. The second entry,
+#'   `"estimated"` is a vector of booleans indicating whether or not
+#'   standard deviation is estimated.
+#' @export
+new_data_distribution <- function(
+  module,
+  family,
```

Comment:
It's unclear if the `family` input can accept multiple options. The user might want to provide a few family options corresponding to a vector of data_type (e.g., `c("index", "agecomp", "lengthcomp")`)? Including a `\dontrun{}` example and specifying the expected dimensions of the input arguments in the Roxygen documentation could help clarify this.

Reply:
added documentation to clarity dimensions of input